src/ImageSharp/Image/PixelAccessor{TColor}.cs

```diff
@@ -186,8 +186,7 @@ public void CopyFrom(PixelArea<TColor> area, int targetY, int targetX = 0)
             int width = Math.Min(area.Width, this.Width - targetX);
             int height = Math.Min(area.Height, this.Height - targetY);
 
-            this.CheckDimensions(width, height);
-
+            // this.CheckDimensions(width, height); TODO: Why was width == 0 or height == 0 considered a problem? Copy implementations do not fail on this (just do nothing)!
```

Comment:
@dlemstra @JimBobSquarePants  regarding `CheckDimensions`:
[I made this hack](https://github.com/antonfirsov/ImageSharp/blob/jpeg-optimizations/src/ImageSharp/Formats/Jpg/Utils/JpegUtils.cs#L47) to conform to the current dimension-checking API definition of `CopyTo()`.
It's like if you defined `Array.Copy()` to throw exception with `Array.Copy(src, dest, 0)`. There are reasons .NET guys have not chosen this definition :)